api/proxy/proxy.go

```diff
@@ -126,6 +131,12 @@ func New(opts *Options) (http.Handler, error) {
 	webProxy.Transport = t
 	webCanaryProxy.Transport = t
 
+	versionRegex, err := regexp.Compile(`mirall/(\d+)\.(\d+).?(\d+)?`)
```

Comment:
What means the first ??

`mirall/(\d+)\.(\d+).?(\d+)?`

Reply:
I wasn't sure we would always get `major.minor.build`, so I decided to allow only `major.minor`. Hence the `?`/

Reply:
Good